architecture description/core/plugins/org.polarsys.kitalpha.ad.integration.sirius/src/org/polarsys/kitalpha/ad/integration/sirius/listeners/ViewpointActivationStateListener.java

```diff
@@ -93,12 +91,9 @@ protected void handleOpenSession(final Session session) {
           newDeselectedViewpoints.add(vp);
         }
       }
-      for (String name : toActivate) {
-        newSelectedViewpoints.add(allSiriusViewpoints.get(name));
```

Comment:
Here some of the viewpoints present in _newSelectedViewpoints_ are not present in the registry. So the general idea is only to base the comparison between activated viewpoints activated in the session and the list of viewpoints referenced in _vp_ models

Reply:
After asking for some support from the Sirius team itself, the state where a Sirius viewpoint is activated but not registered in the ViewpointRegistry is not supported.
I will remove these modifications and add a guard to prevent the _null_ value to be added in the _newSelectViewpoints_ variable.